src/applications/vaos/appointment-list/components/AppointmentsPage/ScheduleNewProjectCheetah.jsx

```diff
@@ -29,7 +29,7 @@ export default function ScheduleNewProjectCheetah({
       <Link
         id="new-project-cheetah"
         className="usa-button vads-u-font-weight--bold vads-u-font-size--md"
-        to="/new-project-cheetah-booking"
+        to="/new-covid-19-booking"
```

Comment:
This is pretty nitpicky, but what about `new-covid-19-vaccine-booking` instead? Just to be extra clear what the booking is for.

Reply:
Done